src/service/fixed-layer.js

```diff
@@ -324,6 +310,44 @@ export class FixedLayer {
   }
 
   /**
+   * Calls `setupFixedSelectors` in a try-catch.
+   * Fails quietly with a dev error if call fails.
+   * This method should not be inlined to prevent TryCatch deoptimization.
+   * NoInline keyword at the end of function name also prevents Closure compiler
+   * from inlining the function.
+   * @param {!Array<string>} fixedSelectors
+   * @private
+   */
+  trySetupFixedSelectorsNoInline(fixedSelectors) {
```

Comment:
No inline is much more self documenting. For the time being there is nothing we can do except choose between not inlining and not obfuscating and obfuscating but inlining.